copr-builder.py

```diff
@@ -373,12 +375,12 @@ def _watch_builds(self, builds):
                     logging.info('Build of %s-%s (ID: %s) finished: %s',
                                  b.package_name, b.package_version, b.id, b.state)
                     if b.state == BuildStateValues.FAILED:
-                        success = False
+                        self._success = False
                     builds.remove(build)
 
             time.sleep(0.5)
 
-        return success
+        return self._success
```

Comment:
Looks like if anything fails, ``self._success`` is never set to ``True`` again and thus all other calls to functions will fail too.